ground/gcs/src/plugins/sysalarmsmessaging/sysalarmsmessagingplugin.cpp

```diff
@@ -93,7 +93,7 @@ void SysAlarmsMessagingPlugin::updateAlarms(UAVObject *systemAlarm)
 {
     UAVObjectField *field = systemAlarm->getField(QString("Alarm"));
 
-    for (uint i = 0; i < field->getNumElements(); ++i) {
```

Comment:
Is this use of 'auto' a win for loop iterators?   In this kind of use it's synonymous with 'int' (e.g. take the type of 0, ignore the other use types).

I mean I guess it's OK because 32 bits is almost always wide enough-- though there is also the whole signed vs. unsigned compare issue.